docs/dask_integration.md

```diff
@@ -58,4 +58,12 @@ Now your `Dataset` consists of Dask arrays, and the client has been set up, all
 ```python
 data_resampled = lilio.resample(calendar, ds)
 ```
+
+Note that this will (quite quickly) return a lazy Dataset: the computation has not been performed yet.
+To compute the result do:
+```python
+data_resampled.compute()
```

Comment:
```suggestion
data_resampled_results = data_resampled.compute()
```
It would be nice to remind the user that the `.compute()` method will not modify the `data_resampled` in-place but instead return another value which contains the resampled results.

Also if the user check the values before executing `data_resampled.compute()`, for instance, call `data_resampled.sst.values` (assuming that we are resampling a dataset with sst), then the computation will be triggered, too. But this is something related to the knowledge of `dask`, may not necessary to add.